projects/developer_docs/resources/automated-user-iam.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_iam_user" "developer-docs-user" {
+    name = "${var.bucket_name}_user"
+}
+
+data "template_file" "developer_docs_automated_read_write" {
+    template = "${file("projects/developer_docs/resources/templates/read_write_user.tpl")}"
+
+    vars {
+        bucket_name = "${var.bucket_name}"
+        environment = "${var.environment}"
```

Comment:
Having an `owner` tag would be nice so we know which team to talk to about this in the future.